packages/sample-app/.env.example

```
@@ -1,6 +1,11 @@
 OPENAI_API_KEY=sk-your-openai-api-key-here
 
+# Atlas Cloud Configuration
+ATLASCLOUD_API_KEY=apikey-your-atlascloud-api-key-here
+ATLASCLOUD_BASE_URL=https://api.atlascloud.ai/v1
+ATLASCLOUD_MODEL=deepseek-ai/DeepSeek-V3-0324
+
 # Traceloop Configuration (Optional - for observability)
 TRACELOOP_API_KEY=your-traceloop-api-key-here
-TRACELOOP_API_ENDPOINT=https://api.traceloop.com
\ No newline at end of file
+TRACELOOP_API_ENDPOINT=https://api.traceloop.com
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The TRACELOOP_API_ENDPOINT key should go before the TRACELOOP_API_KEY key

(UnorderedKey)